setup.py

```diff
@@ -21,7 +19,7 @@ def read(fname):
     name="pinecone-client",
     version=read("pinecone/__version__").strip(),
     description="Pinecone client and SDK",
-    license="Proprietary License",
+    license='Apache 2.0',
```

Comment:
Seems like you need to run black again. It favors double quotes.

Reply:
i don't think black catches the setup.py might need to tweek that